src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,265 @@
+import json
+
+from py42.clients import BaseClient
+from py42.clients.util import get_all_pages
+
+
+class HighRiskEmployeeClient(BaseClient):
+    """
+    Administrator utility to manage High Risk employees.
+    `Support Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Detection_list_management_APIs>`__
+
+    """
+
+    # In other modules URI is a single variable, breaking it up to have better control
+    # if we want to refactor version and prefix to a common configuration.
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}".format(_api_version)
+
+    def __init__(self, session, user_context):
+        super(HighRiskEmployeeClient, self).__init__(session)
+        self._user_context = user_context
```

Comment:
pass in a `DetectionListUserClient` here.

Reply:
Done.